### PR TITLE
Create app dec2020 material icons

### PR DESCRIPTION
**Pull request by AleziaKurdis**
**Material icons in edit mode**

1- This adds a new **Material icon** to identify the material entities that are linked to a parent.
Since those material entities become invisible, this helps to figure their presence.

image

2- The Create App - Preference: "**Show Zones in Create Mode**"
has been removed, since it was doing nothing for a couple of years now.
the method behind is still in the API, but do nothing. 
(abandoned since it was only drawing a box that wasn't useful to figure the shape of the zone)
There was no real value to bring this back in a different way.

3- The Create App - Preference: "**Show Lights and Particle Systems in Create Mode**"
has been **renamed** for "**Show Icons in Create Mode**"
since this setting was also managing all the icons (including Zones and now the Parented Materials)
The objective of the setting is mainly to simplify the display at the edit when a scene is very dense.
(In the future, we might want to make this "à la carte". For now, I think we can try as is. )

4- **3dImage Overlays** used for the **icons** has been migrated to **Local Entities**.

5- All the **Entities icons** (in-world non physical entitities) has been updated to be more **uniformized**. They are now **all with the same dark gray tone**, **with a white border** to keep them visible under different lighting.

### CODE REVIEW

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR907/Vircadia-Alpha-PR907-35e3e9f.tar_28e53c9c.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR907/Vircadia-Alpha-PR907-35e3e9f.tar_28e53c9c.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR907/Vircadia-Alpha-PR907-35e3e9f_28e53c9c.sh

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR907/Vircadia-Alpha-PR907-35e3e9f_28e53c9c.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR907/Vircadia-Alpha-PR907-35e3e9f_28e53c9c.exe

File: scripts/system/libraries/entityIconOverlayManager.js [hunk 1]
```diff
@@ -75,7 +75,11 @@ EntityIconOverlayManager = function(entityTypes, getOverlayPropertiesFunc) {
     function getOverlay() {
         var overlay;
         if (unusedOverlays.length === 0) {
-            overlay = Overlays.addOverlay("image3d", {});
+            overlay = Entities.addEntity({
+                "type": "Image", 
+                "billboardMode": "full", 
+                "emissive": true
+            },"local");
```
Comment (ctrlaltdavid) [inline scripts/system/libraries/entityIconOverlayManager.js:82]:
```suggestion
            overlay = Entities.addEntity({
                type: "Image", 
                billboardMode: "full", 
                emissive: true
            }, "local");
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR907/Vircadia-Alpha-PR907-e9d219c.tar_7ff8236d.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR907/Vircadia-Alpha-PR907-e9d219c.tar_7ff8236d.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR907/Vircadia-Alpha-PR907-e9d219c_7ff8236d.sh

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR907/Vircadia-Alpha-PR907-e9d219c_7ff8236d.dmg

**build (ubuntu-18.04, android)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR907/Vircadia-Alpha-PR907-e9d219c_7ff8236d.apk
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR907/Vircadia-Quest-Alpha-PR907-e9d219c_7ff8236d.apk

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR907/Vircadia-Alpha-PR907-e9d219c_7ff8236d.exe

**Review by ctrlaltdavid — APPROVED** [review]

**Comment by SilverfishVR** [comment]
Looks good to me 👍 
Thank you, I've been wanting this since I started using material entities 😃

**Comment by Aitolda** [comment]
Seems to be working as intended.

**Pull request merged**